components/theia/packages/gitpod-extension/src/node/gitpod-backend-module.ts

```diff
@@ -20,7 +20,6 @@ import { ServedPortsServiceClient, ServedPortsService, ServedPortsServiceServer
 import { GitpodEnvVariablesServer } from "./gitpod-env-variables-server";
 import { ShellProcess } from "@theia/terminal/lib/node/shell-process";
 import { GitpodShellProcess } from "./gitpod-shell-process";
-import { ServedPortsServiceServerImpl } from "./served-ports-service-server";
```

Comment:
`served-ports-service-server` should be deleted?

Reply:
I guess it could go altogether. Somehow I was too timid to remove it. Same goes for the `isPortReady` and `waitUntilPortIsReady` functions - they're just needed for the old implementation. I'll remove the old stuff completely.